docs/pages/docs/custom-schemas/custom-blocks.mdx

```diff
@@ -77,76 +41,108 @@ const Alert = createReactBlockSpec(
 );
 ```
 
-You can see that `createReactBlockSpec` takes two object arguments:
+#### Block Config (`CustomBlockConfig`)
 
-#### `blockConfig`
+The Block Config describes the shape of the Block, it's type and the properties it supports:
 
-This defines the block's type, properties, and content type. It allows BlockNote to know how to handle manipulating the block internally and provide typing.
+```typescript
+type BlockConfig = {
+  type: string;
+  content: "inline" | "none";
+  readonly propSchema: PropSchema;
+};
+```
 
 **`type`**
```

Comment:
Not relevant for now, but at some point we should probably remove the `type` field since it's just a duplicate of the key in the schema